### PR TITLE
format filenames in README

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

**Comment by navidcy** [comment]
@josuemtzmo review?

File: README.rst [hunk 1]
```diff
@@ -12,9 +12,9 @@ Each folder included within this repository has the following structure:
 
 - Example Name
  
-  * example_blender_file.blend
-  * example_animation.png
-  * example_animation.rst
+  * ``example_blender_file.blend``
+  * ``example_animation.png``
+  * ``example_animation.rst``
```
Comment (josuemtzmo) [inline README.rst:20]: Now that you have done this, could you add the following two lines at the end?  
```
  * ``example_animation.mp4``
  * ``example_animation.mov``
```

**Review by josuemtzmo — CHANGES_REQUESTED** [review]

File: README.rst [hunk 2]
```diff
@@ -12,9 +12,9 @@ Each folder included within this repository has the following structure:
...
```
Comment (navidcy) [inline README.rst:17]:
```suggestion
  * ``example_animation.rst``
  * ``example_animation.mp4``
  * ``example_animation.mov``
```

**Review by josuemtzmo — APPROVED** [review]
Awesome!

**Pull request merged**